backend/src/EntityExtractor.Contracts/IFileStoring.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace eKultura.EntityExtractor.Contracts;
+
+public interface IFileStoring
```

Comment:
There will be two man use cases related to this class:
- file upload (save)
- file download (get)

Based in that, I would suggest renaming it to either IFileStorage or IFileStoringService.